dnf-testing.sh

```diff
@@ -110,8 +110,10 @@ run()
     [ ${#TESTS[@]} -eq 0 ] && TESTS=("${FEATURES[@]}")
     local failed=0
     for feature in "${TESTS[@]}"; do
-        sudo docker run --rm "$IMAGE" launch-test "$feature" dnf >&2 || :
-        [ $? -ne 0 ] && let ++failed
+        for command in dnf-2 dnf-3; do
```

Comment:
basically it will run dnf2 and dnf3 in one container which will double time for testing. We should spawn 2 containers insted.